language/stdlib/modules/LibraSystem.move

```diff
@@ -70,6 +73,11 @@ module LibraSystem {
     const EINVALID_TRANSACTION_SENDER: u64 = 4;
     /// An out of bounds index for the validator set was encountered
     const EVALIDATOR_INDEX: u64 = 5;
+    /// Rate limited when trying to update config
+    const ECONFIG_UPDATE_RATE_LIMITED: u64 = 6;
+
+    /// Number of microseconds in 5 minutes
+    const FIVE_MINUTES: u64 = 300000;
```

Comment:
If this really is five minutes in microseconds, shouldn't this be 300000000?

Reply:
True, thank you so much for catching this!